<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocToDirDialog</class>
 <widget class="QDialog" name="DocToDirDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Put Document to Directory</string>
  </property>
  <widget class="QWidget" name="DocToDirWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>-10</y>
     <width>102</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="docToDirBtn">
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="locale">
       <locale language="Chinese" country="China"/>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="newDirBtn">
      <property name="text">
       <string>新建文件夹</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>9</y>
     <width>341</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="docToDirTree">
    <item>
     <widget class="QLabel" name="sourceDoc">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DocToDirDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
